docs/index.md

```diff
@@ -0,0 +1 @@
+# DerivativeFreeSolvers.jl
```

Comment:
```suggestion
# DerivativeFreeSolvers.jl

Solvers that are derivative-free. Docs in progress.
```

Reply:
This can’t really be done in a generic way across all packages. Doc templates are minimalistic.